doc/examples/plot_entropy.py

```diff
@@ -33,40 +28,47 @@
 from skimage.filters.rank import entropy
 from skimage.morphology import disk
 
+# First example: object detection.
+
 noise_mask = 28 * np.ones((128, 128), dtype=np.uint8)
 noise_mask[32:-32, 32:-32] = 30
 
-noise = (noise_mask * np.random.random(noise_mask.shape) - .5 *
+noise = (noise_mask * np.random.random(noise_mask.shape) - 0.5 *
          noise_mask).astype(np.uint8)
 img = noise + 128
 
-radius = 10
-e = entropy(img, disk(radius))
+entr_img = entropy(img, disk(10))
+
+fig, ax = plt.subplots(1, 3, figsize=(8, 3))
+ax0, ax1, ax2 = ax.ravel()
```

Comment:
Could just use (ax0, ax1, ax2) in prior line. I'm not sure this is a standard for us, but it seems to be common (and the form used later).